docs/docs/behaviors/hold-tap.md

```diff
@@ -83,6 +83,10 @@ For example, if you press `&mt LEFT_SHIFT A` and then release it without pressin
 };
 ```
 
+#### `hold-while-undecided`
+
+If enabled, the hold behavior will immediately be held, and will release before the tap behavior is sent. With modifiers (excluding the windows key) this will not affect typing, and is useful for mod + clicking.
```

Comment:
Note that `alt` modifiers can also do things like activate menu hotkeys when used on their own. Also worth noting this and `quick-tap` are exclusive (right)?

Reply:
A quick tap will never reach the hold-while-undecided code and vice-versa, so you can use them both on the same behavior.  Good thinking about the alt behavior, I quickly found out that part out.